thêm logic hạn chế di chuyển crop layer không hợp lý

vvt2001 opened the pull request.

vvt2001 committed:
thêm logic hạn chế di chuyển crop layer không hợp lý

linhndSolar commented on Crop Image/CroppingLayerView.swift:
```diff
@@ -18,23 +18,45 @@ class CroppingLayerView: UIView {
     private var bottomRightCornerIndicatorView = CornerIndicatorView()
     private var bottomLeftCornerIndicatorView = CornerIndicatorView()
     
+    private func limitedCroppingLayer(frame: CGRect) -> CGRect {
```
Hàm trả ra CGRect, nhưng tên hàm lại không thể hiện được điều đó

vvt2001 committed:
sửa tên hàm

vvt2001 committed:
clean code

định sẵn giá trị kích cỡ indicator và thay các giá trị số theo tỉ lệ kích cỡ đó

vvt2001 committed:
clean code

cho phần kích cỡ của indicator trong phần khởi tạo, hàm update chỉ update x y
